client/src/modules/dashboard/Events/pages/EditEventPage.tsx

```diff
@@ -67,9 +70,17 @@ export const EditEventPage: NextPage = () => {
       });
 
       if (event.data) {
+        toast({
+          title: `Event "${event.data.updateEvent.name}" updated successfuly!`,
+          status: 'success',
+        });
         await router.push('/dashboard/events');
```

Comment:
```suggestion
        await router.push('/dashboard/events');
        toast({
          title: `Event "${event.data.updateEvent.name}" updated successfuly!`,
          status: 'success',
        });
```
What do you think about swapping order here? Right now the appearing animation happens twice, after clicking save button when data is saved and after opening `/dashboard/events`. With swapped order toast appears just in the dashboard.

Reply:
I could not notice this on my end :/. Anyway, I made the suggested change.